### PR TITLE
OSSMDOC-472: Make NetworkPolicy optional.

### PR DESCRIPTION
**Pull request by JStickler**
This PR adds two new topics to the Traffic Management assembly.

Preview is here ->https://deploy-preview-41402--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-traffic-manage.html#ossm-understanding-networkpolicy_routing-traffic

Eng review - rcernich, jwendell
QE review - yxun
Peer review - ahardin-rh

### CODE REVIEW

File: modules/ossm-config-disable-networkpolicy.adoc [hunk 1]
```diff
@@ -15,11 +13,16 @@ If you want to disable the automatic creation and management of `NetworkPolicies
...
+
+* {ProductName} Operator version 2.1.1 or higher installed.
+* `ServiceMeshControlPlane` resource updated to version 2.1.1 or higher.
+
 .Procedure
 
 . In the {product-title} web console, click *Operators* -> *Installed Operators*.
 
-. Select the project where you installed the control plane, for example `istio-system`, from the Project menu.
+. Select the project where you installed the control plane, for example `istio-system`, from the *Project* menu.
```
Comment (jwendell) [inline modules/ossm-config-disable-networkpolicy.adoc:25]: Remove the `trust:` line in the snippet below.

File: modules/ossm-config-disable-networkpolicy.adoc [hunk 2]
```diff
@@ -15,11 +13,16 @@ If you want to disable the automatic creation and management of `NetworkPolicies
 When you disable `spec.security.manageNetworkPolicy` {ProductName} will not create *any* `NetworkPolicy` objects.  The system administrator is responsible for managing the network and fixing any issues this might cause.
 ====
 
+.Prerequisites
+
+* {ProductName} Operator version 2.1.1 or higher installed.
+* `ServiceMeshControlPlane` resource updated to version 2.1.1 or higher.
```
Comment (jwendell) [inline modules/ossm-config-disable-networkpolicy.adoc:19]: SMCP's version is in the form `v2.0`, `v2.1`, etc. So this statement could be reworded to something like:

`ServiceMeshControlPlane` resource updated to version `v2.1`.

File: modules/ossm-config-disable-networkpolicy.adoc [hunk 3]
```diff
@@ -4,9 +4,7 @@ This module included in the following assemblies:
 ////
 
 [id="ossm-config-disable-networkpolicy_{context}"]
-= Disabling network policies
-
-{ProductName} automatically creates and manages a number of `NetworkPolicies` resources in the control plane and application namespaces. This is to ensure that applications and the control plane can communicate with each other.
+= Disabling automatic creation of network policies
 
 If you want to disable the automatic creation and management of `NetworkPolicies` resources, for example to enforce company security policies, you can do so.  You can edit the `ServiceMeshControlPlane` to set the `spec.security.manageNetworkPolicy` setting to `false`
```
Comment (rcernich) [inline modules/ossm-config-disable-networkpolicy.adoc:9]:
```suggestion
If you want to disable the automatic creation and management of `NetworkPolicy` resources, for example to enforce company security policies, or to allow direct access to pods in the mesh, you can do so.  You can edit the `ServiceMeshControlPlane` and set `spec.security.manageNetworkPolicy` to `false`
```

**Review by rcernich — APPROVED** [review]

**Review by jwendell — APPROVED** [review]

**Review by yxun — APPROVED** [review]

**Comment by ahardin-rh** [comment]
LGTM

**Pull request merged**